### PR TITLE
Added new rc rate calculation to rates curve. Removed SUPER_EXPO feature for >= 3.0

### PR DESCRIPTION
**Pull request by borisbstyle**
continue work from #252

### CODE REVIEW

File: tabs/pid_tuning.html [hunk 1]
```diff
@@ -66,13 +66,14 @@
...
                         <table class="pid_titlebar">
                             <tr>
-                                <th class="name" i18n="pidTuningName"></th>
-                                <th class="proportional" i18n="pidTuningProportional"></th>
-                                <th class="integral" i18n="pidTuningIntegral"></th>
-                                <th class="derivative" i18n="pidTuningDerivative"></th>
-                                <th class="rc_rate" i18n="pidTuningRcRate"></th>
-                                <th class="rate" i18n="pidTuningRate"></th>
-                                <th class="rc_expo" i18n="pidTuningRcExpo"></th>
+                                <th class="name" i18n="pidTuningName" style="width: 12.5%;"></th>
```
Comment (mikeller) [inline tabs/pid_tuning.html]: @borisbstyle: Can you please revert this? No point totally messing up the html and only creating more work for @skaman82

File: tabs/pid_tuning.html [hunk 2]
```diff
@@ -66,13 +66,14 @@
...
-                                <th class="rate" i18n="pidTuningRate"></th>
-                                <th class="rc_expo" i18n="pidTuningRcExpo"></th>
+                                <th class="name" i18n="pidTuningName" style="width: 13%;"></th>
+                                <th class="proportional" i18n="pidTuningProportional" style="width: 11%;"></th>
+                                <th class="integral" i18n="pidTuningIntegral" style="width: 13%;"></th>
+                                <th class="derivative" i18n="pidTuningDerivative" style="width: 12%;"></th>
+                                <th class="rc_rate" i18n="pidTuningRcRate" style="width: 13%;"></th>
+                                <th class="rate" i18n="pidTuningRate" style="width: 13%;"></th>
+                                <th class="rate" i18n="pidTuningMaxVel" style="width: 12%;"></th>
+                                <th class="rc_expo" i18n="pidTuningRcExpo" style="width: 13%;"></th>
```
Comment (mikeller) [inline tabs/pid_tuning.html]: Styling should go into the .css, and not into the html.

File: tabs/pid_tuning.css [hunk 3]
```diff
@@ -240,7 +240,7 @@
 
 .tab-pid_tuning .pid_titlebar th {
     padding: 5px;
-    text-align: left;
```
Comment (mikeller) [inline tabs/pid_tuning.css]: This is inconsistent with pretty much every other table in the app.

File: tabs/pid_tuning.css [hunk 4]
```diff
@@ -370,32 +370,8 @@
     border-top-right-radius: 3px;
 }
 
-.tab-pid_tuning .name {
```
Comment (mikeller) [inline tabs/pid_tuning.css]: Why are you removing these? They are needed.

File: tabs/pid_tuning.css [hunk 5]
```diff
@@ -370,32 +370,8 @@
...
-}
-
-.tab-pid_tuning .rate {
-    width: 14%;
-}
-
-.tab-pid_tuning .rc_expo {
-    width: 14%;
+.tab-pid_tuning .new_rates {
+    text-align: center;
```
Comment (mikeller) [inline tabs/pid_tuning.css]: This is not consistent with the layout of the rest of the app.

File: tabs/pid_tuning.html [hunk 6]
```diff
@@ -66,14 +66,14 @@
...
                             <tr>
-                                <th class="name" i18n="pidTuningName" style="width: 13%;"></th>
-                                <th class="proportional" i18n="pidTuningProportional" style="width: 11%;"></th>
-                                <th class="integral" i18n="pidTuningIntegral" style="width: 13%;"></th>
-                                <th class="derivative" i18n="pidTuningDerivative" style="width: 12%;"></th>
-                                <th class="rc_rate" i18n="pidTuningRcRate" style="width: 13%;"></th>
-                                <th class="rate" i18n="pidTuningRate" style="width: 13%;"></th>
-                                <th class="rate" i18n="pidTuningMaxVel" style="width: 12%;"></th>
-                                <th class="rc_expo" i18n="pidTuningRcExpo" style="width: 13%;"></th>
+                                <th class="name" i18n="pidTuningName" style="width: 12.5%;"></th>
```
Comment (mikeller) [inline tabs/pid_tuning.html]: This is done in the wrong place, should be done in the .css.

File: tabs/pid_tuning.html (hunk 2 above)
Comment (TheAngularity) [inline tabs/pid_tuning.html]: titlebar from accel edited the same way

File: tabs/pid_tuning.css (hunk 3 above)
Comment (TheAngularity) [inline tabs/pid_tuning.css]: titlebar from accel have also text align centered

File: tabs/pid_tuning.css (hunk 4 above)
Comment (TheAngularity) [inline tabs/pid_tuning.css]: has not been used anymore

File: tabs/pid_tuning.css (hunk 5 above)
Comment (TheAngularity) [inline tabs/pid_tuning.css]: but looks better as pressed on the left side

File: tabs/pid_tuning.html (hunk 1 above)
Comment (TheAngularity) [inline tabs/pid_tuning.html]: see #259

File: tabs/pid_tuning.css (hunk 3 above)
Comment (TheAngularity) [inline tabs/pid_tuning.css]: yep, see it ... pls do not merge this!!!

**Pull request merged**